data-processing/entities/abbreviations/types.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+from common.types import SerializableEntity
+
+
+@dataclass(frozen=True)
+class Abbreviation(SerializableEntity):
+    text: str
+    expansion : str
+    #the str_id tells us whether it is a full form or short form (f or s),
```

Comment:
Ah I see! 'f' is full and 's' is short. Can this just be 'full' and 'short'? The meaning of this property may also be clearer with a name like `str_type` or `abbr_pair_type`

Reply:
I thought we wanted it all in one string, but it can broken up if preferred.